### ISSUE TITLE
Set up a website and make a new release

### ISSUE THREAD

**Issue by alex-konovalov** [gap-packages/gbnp#2]
For packages hosted on https://github.com/gap-packages, we offer some tools which relieve technical burden of making a release and remembering all steps in the right order. First, they automate the procedure; Second, someone who has necessary permissions may be able to publish a release for packages authors. The tools are
* https://github.com/gap-system/ReleaseTools 
* https://github.com/gap-system/GitHubPagesForGAP

and their documentation explains how to set them up. We will be happy to set this up for GBNP.

**Comment by alex-konovalov** [comment]
One of the problems this will fix is the missing `.css` file in the distribution which makes the online version of the manual at https://www.gap-system.org/Manuals/pkg/gbnp/doc/chap0.html less convenient to read.

**Comment by alex-konovalov** [comment]
For the record, the old location of GBNP is not available any more:
```
  WARNING (GBNP): no success in download of the current info file 
  from http://mathdox.org/products/gbnp/PackageInfo.g
```

**Comment by alex-konovalov** [comment]
@jwknopper Was checking package update logs - the only package for which we can't check for updates (because its old website does not work any more) is GBNP. What could we do to make some progress - this issue is now 922 days old :(

**Comment by fingolfin** [comment]
I have just released GBNP 1.0.4.
